src/php/Ast/PhpVarNode.php

```diff
@@ -33,7 +33,7 @@ final class PhpVarNode extends Node
         '**',
         '^',
         '<<',
-        '>>'
+        '>>',
```

Comment:
I think we should define a rule, if there a comment at the end of the array or not ;)
What do you think?

Reply:
I agree. I would vote 👍 
There is a phpcs for it: [trailing_comma_in_multiline_array](https://mlocati.github.io/php-cs-fixer-configurator/#version:2.15|fixer:trailing_comma_in_multiline_array)